mbuild/box.py

```diff
@@ -3,179 +3,433 @@
+
+    @property
+    def xy(self):
+        """Tilt factor xy of the box."""
+        return round(self._xy, self.precision)
+
+    @property
+    def xz(self):
+        """Tilt factor xz of the box."""
+        return round(self._xz, self.precision)
+
+    @property
+    def yz(self):
+        """Tilt factor yz of the box."""
+        return round(self._yz, self.precision)
+
+    @property
+    def tilt_factors(self):
+        """Return the 3 tilt_factors (xy, xz, yz) of the box."""
+        return self.xy, self.xz, self.yz
 
     @property
     def angles(self):
-        """Get the angles of the box."""
-        return self._angles
-
-    @mins.setter
-    def mins(self, mins):
-        mins = np.array(mins, dtype=float)
-        assert mins.shape == (3,)
-        assert all(mins <= self.maxs), "Given mins is greater than maxs"
-        self._mins = _BoxArray(array=mins, var="mins", box=self)
-        self._lengths = _BoxArray(
-            array=(self.maxs - self.mins), var="lengths", box=self
-        )
+        """Angles defining the tilt of the box (alpha, beta, gamma)."""
+        (alpha, beta, gamma) = self._get_angles()
+        alpha = round(alpha, self.precision)
+        beta = round(beta, self.precision)
+        gamma = round(gamma, self.precision)
+        return alpha, beta, gamma
 
-    @maxs.setter
-    def maxs(self, maxs):
-        maxs = np.array(maxs, dtype=float)
-        assert maxs.shape == (3,)
-        assert all(maxs >= self.mins), "Given maxs is less than mins"
-        self._maxs = _BoxArray(array=maxs, var="maxs", box=self)
-        self._lengths = _BoxArray(
-            array=(self.maxs - self.mins), var="lengths", box=self
-        )
+    @property
+    def precision(self):
+        """Amount of decimals to represent floating point values."""
+        return self._precision
 
-    @lengths.setter
-    def lengths(self, lengths):
-        if isinstance(lengths, int) or isinstance(lengths, float):
-            lengths = np.array(lengths * np.ones(3), dtype=float)
+    @precision.setter
+    def precision(self, value):
+        """Decimal point precision, if None use 16, else cast as int."""
+        if not value:
+            precision = 16
         else:
-            lengths = np.array(lengths, dtype=float)
-        assert lengths.shape == (3,)
-        assert all(lengths >= 0), "Given lengths are negative"
-        self._maxs = _BoxArray(
-            array=(self.maxs + (0.5 * lengths - 0.5 * self.lengths)),
-            var="maxs",
-            box=self,
+            precision = int(value)
+        self._precision = precision
+
+    @property
+    def bravais_parameters(self):
+        """Return the Box representation as Bravais lattice parameters.
+
+        Based on the box vectors, return the parameters to describe the box in
+        terms of the Bravais lattice parameters:
+            a,b,c = the edges of the Box
+            alpha, beta, gamma = angles(tilt) of the parallelepiped, in degrees
```

Comment:
```suggestion
        terms of the Bravais lattice parameters:
        
            a,b,c = the edges of the Box
            alpha, beta, gamma = angles(tilt) of the parallelepiped, in degrees
```
I think this is throwing a warning in the read the docs build